packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -975,4 +978,83 @@ test('receiveUpcall returns false if sender is not AXELAR_GMP', async t => {
   t.is(upcallProcessed, false, 'upcall indicates bad GMP sender');
 });
 
+test('handle failure in provideCosmosAccount makeAccount', async t => {
+  const amount = make(USDC, 100n);
+  const { orch, ctx, offer } = mocks(
+    { makeAccount: Error('Insufficient funds for account creation') },
```

Comment:
We need to show that things work after they fail.

Failure to create an agoric account is implausible. Let's test failure to create @noble.

```suggestion
  const chainToErr = new Map(['noble', Error('timeout creating ICA')]);
  
  const { orch, ctx, offer } = mocks(
    { makeAccount: chainToErr },
```